### PR TITLE
Multi-arch Images?

### PR DESCRIPTION
**Pull request by mannkind**
First: Thanks for Miniflux. I simply love it.
Second: I have no idea if you're interested in multi-arch images; I recognize this might only be important to me :)

I've tried to adapt the techniques found at [building multi-arch docker images](https://lobradov.github.io/Building-docker-multiarch-images) to your Makefile. 

I'm not an expert in any of this and I don't have the hardware at the moment to test arm64v8, but it appears to be working for amd64 and arm32v6 (on a Tinkerboard); tested with Miniflux 2.0.8-2.0.11.

[Example images](https://hub.docker.com/r/mannkind/miniflux/tags/)

### CODE REVIEW

File: Dockerfile.cross [hunk 1]
```diff
@@ -0,0 +1,19 @@
+FROM golang:1.11-alpine3.8 as build
+ENV GO111MODULE=on
+RUN apk add --no-cache --update build-base git && \
+    mkdir -p /go/src/github.com/miniflux/ && \
+    cd /go/src/github.com/miniflux/ && \
+    git clone https://github.com/miniflux/miniflux.git miniflux && \
+    cd miniflux && \
+    git checkout __MINIFLUX_VERSION__ && \
+    make clean && \
+    go get && \
```
Comment (fguillot) [inline Dockerfile.cross]: It should work without the `go get` because dependencies are vendored?

File: Makefile [hunk 2]
```diff
@@ -1,12 +1,36 @@
...
+	for arch in $(ARCHS); do \
+	  case $${arch} in \
+		amd64   ) qemu_arch="x86_64"; golang_arch="amd64"; miniflux_arch="amd64";; \
+		arm32v6 ) qemu_arch="arm"; golang_arch="arm"; miniflux_arch="armv6";; \
+		arm64v8 ) qemu_arch="aarch64"; golang_arch="arm64"; miniflux_arch="armv8";; \
+	  esac ;\
+	  cp Dockerfile.cross Dockerfile.$${arch} ;\
+	  sed -i "" "s|__BASEIMAGE_ARCH__|$${arch}|g" Dockerfile.$${arch} ;\
+	  sed -i "" "s|__QEMU_ARCH__|$${qemu_arch}|g" Dockerfile.$${arch} ;\
+	  sed -i "" "s|__GOLANG_ARCH__|$${golang_arch}|g" Dockerfile.$${arch} ;\
```
Comment (fguillot) [inline Makefile]: Why do you need the `__QEMU_ARCH__` and the `__GOLANG_ARCH__ `, it doesn't seems to be defined elsewhere?

**Comment by fguillot** [comment]
I tested the `arm64v8` image, seems to work fine. I will push the new images for the next release.

Thanks!

**Comment by mannkind** [comment]
Awesome. I see you've already further improved the setup.

This is exciting; I look forward to your next release!

**Pull request merged**